api/api.py

```diff
@@ -44,13 +50,22 @@ def too_large(e):
 
 
 class RecrudescenceTest(Resource):
+
+    decorators = [
```

Comment:
Where are these decorators actually applied? Quickly checking, this list syntax doesn't seem to be one of the standard ways of applying limits to a flask function (https://flask-limiter.readthedocs.io/en/stable/)

Have you tested to make sure the limiter is actually being used right now?

Reply:
Yes, it has been tested and works properly. If you also want to test it, just turn the max request limit down very low. There were problems with the standard ways outlined in the docs- if you want I can try to find the stackoverflow post i referenced in implementing that line